cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -120,8 +135,11 @@ MapBuilderBridge::BuildOccupancyGrid() {
   if (!trajectory_nodes.empty()) {
     occupancy_grid =
         cartographer::common::make_unique<nav_msgs::OccupancyGrid>();
-    cartographer_ros::BuildOccupancyGrid(trajectory_nodes, options_,
-                                         occupancy_grid.get());
+    cartographer_ros::BuildOccupancyGrid(
```

Comment:
Add the check that was removed from `cartographer_ros::BuildOccupancyGrid()` here? Rename that function to `BuildOccupancyGrid2D` or so?